### PR TITLE
Change/input rework

### PR DESCRIPTION
**Pull request by KiwiLT**
## Description
Input reworked to the new input controls, aligning with designer's input schema. Note that possessing another object when already possessing IS NOT IMPLEMENTED YET. You can try and enter possession aim mode but it will be buggy.

## Setting up testing environment
1. In the project window navigate to Project/Assets/Scenes and open the InputRework scene.
2. Press Play.

#### Testing the Anxiety Meter
Criteria:

- [x] Player enters Possess Aim mode with Left Mouse Press.
- [x] Player cancels Aim mode or Throw mode with c.
- [x] Player confirms aim mode with LMB release.
- [x] Player cancels aim mode with LMB release when not selecting an object.
- [x] Player enters throw mode with RMB.
- [x] Player throws with RMB release.
- [x] Player unpossess with Q.
- [x] Player can enter Possess aim mode while possessing an object.
- [x] Throw line should not be visible in Possess aim mode.

## ToDo
- [ ] Possessing a different object when already possessing needs to be fixed. This includes:
  - [ ] Camera should have a smooth transition.
  - [ ] It should call Unpossess() first before repossessing.

## Checklist
_These checks should always be true for your pull request_
- [x] My pull request is for one story/feature.
- [x] Every individual commit in this pull request is logical.
- [x] All code, documentation, commits and player seen texts are in English.
- [x] I have deleted unused / unnecessary code.
- [x] If my edits need a change in documentation, then I have updated the documentation.
- [x] All code is in correspondence with the code conventions.

### CODE REVIEW

File: Assets/Scripts/Player/AimMode.cs [hunk 1]
```diff
@@ -22,12 +22,17 @@ public class AimMode : MonoBehaviour
     private float _aimSpeed;
 
     public bool aimmode;
```
Comment (ThijsvanDijk) [inline Assets/Scripts/Player/AimMode.cs:24]: Please start public variables with capital letter

File: Assets/Scripts/Player/AimMode.cs [hunk 2]
```diff
@@ -87,7 +93,9 @@ public void EnterAimMode()
     public void ExitAimMode()
```
Comment (hassanmohamed2003) [inline Assets/Scripts/Player/AimMode.cs:93]: why is this function called ExitAimMode if it also does ThrowMode functionality?

**Review by hassanmohamed2003 — CHANGES_REQUESTED** [review]
it works, just one minor point

**Review by ThijsvanDijk — APPROVED** [review]
Looks good!

File: Assets/Scripts/Player/AimMode.cs (hunk 2 above)
Comment (KiwiLT) [reply Assets/Scripts/Player/AimMode.cs:93]: throw mode is a special aim mode

**Pull request merged**